runtime/src/snapshot_utils.rs

```diff
@@ -95,6 +95,7 @@ pub const DEFAULT_MAX_FULL_SNAPSHOT_ARCHIVES_TO_RETAIN: NonZeroUsize =
     unsafe { NonZeroUsize::new_unchecked(2) };
 pub const DEFAULT_MAX_INCREMENTAL_SNAPSHOT_ARCHIVES_TO_RETAIN: NonZeroUsize =
     unsafe { NonZeroUsize::new_unchecked(4) };
+pub const SNAPSHOT_ARCHIVE_DISABLED_INTERVAL: Slot = Slot::MAX;
```

Comment:
for me, `DISABLED_SNAPSHOT_ARCHIVE_INTERVAL` reads a bit better. but not a very strong opinion on it.

Reply:
Done